rasa/nlu/selectors/response_selector.py

```diff
@@ -289,15 +304,29 @@ def __init__(
 
         super().__init__(
             component_config,
+            model_storage,
+            resource,
+            execution_context,
             index_label_id_mapping,
             entity_tag_specs,
             model,
-            finetune_mode=finetune_mode,
             sparse_feature_sizes=sparse_feature_sizes,
         )
 
+    @classmethod
+    def create(
+        cls,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ) -> ResponseSelectorGraphComponent:
+        """Creates a new untrained component (see parent class for full docstring)."""
+        return cls(config, model_storage, resource, execution_context)
+
```

Comment:
there is actually no need for this right? we can just use the one from the parent class
```suggestion
```